server/middleware/rest.js

```diff
@@ -28,10 +29,16 @@ function rest() {
   return function restApiHandler(req, res, next) {
     var app = req.app;
 
-    if (req.url === '/routes') {
-      return res.send(app.handler('rest').adapter.allRoutes());
-    } else if (req.url === '/models') {
-      return res.send(app.remotes().toJSON());
+    // added for https://github.com/strongloop/loopback/issues/1134
+    var legacyExplorer = (undefined === app.get('legacyExplorer') || app.get('legacyExplorer')) ? true : false;
+
+    if (legacyExplorer && 'get' === req.method.toLowerCase()) {
```

Comment:
Please keep support for all methods in order to preserve backwards compatibility, even if it's silly.